### PR TITLE
Elytron subsystem - WIP

### PR DESCRIPTION
**Pull request by claudio4j**
* added two resources: policy, aggregate-security-event-listener

### CODE REVIEW

File: app/src/main/java/org/jboss/hal/client/configuration/subsystem/elytron/OtherSettingsView.java [hunk 1]
```diff
@@ -66,6 +75,9 @@
 
     private OtherSettingsPresenter presenter;
 
+    private static final Constants CONSTANTS = GWT.create(Constants.class);
```
Comment (hpehl) [inline app/src/main/java/org/jboss/hal/client/configuration/subsystem/elytron/OtherSettingsView.java]: We should use static constants for `Constants` and `Messages` only if there's no DI available. Please inject `org.jboss.hal.resources.Resources` instead, which gives you access to all kind of resources (i18n, preview texts, images, ...)

File: app/src/main/java/org/jboss/hal/client/configuration/subsystem/elytron/OtherSettingsView.java [hunk 2]
```diff
@@ -256,6 +258,48 @@
...
 
+        aggregateSecurityEventListenerView = new ResourceView.Builder(tableButtonFactory, primaryIdLogs,
+                Ids.ELYTRON_AGGREGATE_SECURITY_EVENT_LISTENER, "Aggregate Security Event Listener",
+                AGGREGATE_SECURITY_EVENT_LISTENER_ADDRESS, this,
+                () -> presenter.reload())
+                .setNavigation(navigation)
+                .setMetadataRegistry(metadataRegistry)
+                .setTableAddCallback((name, address) -> presenter.reload())
+                .build()
+                .create();
```
Comment (hpehl) [inline app/src/main/java/org/jboss/hal/client/configuration/subsystem/elytron/OtherSettingsView.java]: Why is there a `build()` *and* `create()` method?

File: app/src/main/java/org/jboss/hal/client/configuration/subsystem/elytron/OtherSettingsView.java (hunk 1 above)
Comment (claudio4j) [inline app/src/main/java/org/jboss/hal/client/configuration/subsystem/elytron/OtherSettingsView.java]: Done.

File: app/src/main/java/org/jboss/hal/client/configuration/subsystem/elytron/OtherSettingsView.java (hunk 2 above)
Comment (claudio4j) [inline app/src/main/java/org/jboss/hal/client/configuration/subsystem/elytron/OtherSettingsView.java]: The builder constructs a ResourceView.Builder() class, that is a builder preparation, them the ResourceView.create() creates all necessary table, forms, tabs, pages then add them to layout.

**Comment by hpehl** [comment]
I'm going to merge this. But IMHO I think the ResourceView tries to do a little too much for one class. Does not quite follow the [SoC](https://en.wikipedia.org/wiki/Separation_of_concerns) design principle. Maybe we can refactor this later.

**Pull request merged**